templates/kmmo.yaml

```diff
@@ -495,6 +495,8 @@ spec:
         - mountPath: /controller_config.yaml
           name: manager-config
           subPath: controller_config.yaml
+        - mountPath: /sys/devices/virtual/powercap
+          name: powercapmask
```

Comment:
Out of curiosity, why this is masked out from sysfs mount?

PS. Would some short changelog of updates (between GPU util versions) make sense?

Reply:
> PS. Would some short changelog of updates (between GPU util versions) make sense?

Yep, I could add that.

Reply:
Add a comment with the CVE number?